tests/snp_launch.rs

```diff
@@ -172,27 +69,21 @@ fn snp() {
 
     let finish = SnpFinish::new(None, None, [0u8; 32]);
 
-    let mut vcpu_fd = launcher.as_mut().create_vcpu(0).unwrap();
-    set_cpu(&kvm_fd, &mut vcpu_fd);
+    let vcpu_fd = launcher.as_mut().create_vcpu(0).unwrap();
+
+    let mut regs = vcpu_fd.get_regs().unwrap();
+    regs.rip = MEM_ADDR;
+    regs.rflags = 2;
```

Comment:
Would be nice to add a comment here regarding what this does for people lacking context (like me).
Didn't find anything in Kernel docs and what I understood from https://zserge.com/posts/kvm/ is that this bit should always be set to 1.
If that's the case, how about `0b10` for clarity?